### ISSUE TITLE
[Feature Request] Implement the option to use Text Displays instead of Holograms

### ISSUE THREAD

**Issue by sluhtie** [Pyrbu/ZNPCsPlus#141]
It would be pretty cool if there were an option to toggle text displays usage. This would allow more customisation for the holograms. As example you could remove the background or add text shadows to the displays. Also scaling would be a option.
![grafik](https://github.com/Pyrbu/ZNPCsPlus/assets/97464371/3c92a179-47f6-46ef-9ece-bb21f37dd30a)

**Comment by b0hle** [comment]
I personally do not rely on ZNPCs for managing our holograms due to their current limitations. If NPC holograms offered more customization options and were easier to configure, I would use them extensively.

**Comment by sluhtie** [comment]
> I personally do not rely on ZNPCs for managing our holograms due to their current limitations. If NPC holograms offered more customization options and were easier to configure, I would use them extensively.

I agree, for now I just removed the holograms aswell and made custom text displays. But it would be cool to have such a feature in the future
